packages/components/src/form-token-field/index.js

```diff
@@ -397,14 +402,19 @@ class FormTokenField extends Component {
 	}
 
 	addNewToken( token ) {
+		const { expandOnFocus, validateInput } = this.props;
+		if ( ! validateInput( token ) ) {
+			this.props.speak( this.props.messages.invalid, 'assertive' );
+			return;
+		}
```

Comment:
It looks like the `messages` prop is not documented. It would also be cool to have a story with `validateInput` so we can test with screen readers.

Reply:
I documented the `messages` prop in b9df95d0a4f9531a94f3903f0e23904c23bdf428 and added a knob that enables validation in 581d3e7fa339f7a386e949854b72093bc2dd7ad4.